configs/settings-example.yml

```diff
@@ -0,0 +1,131 @@
+# Variables for ansible provisioning
+# See https://docs.ansible.com/ansible/latest/playbooks_variables.html
+# https://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable
+env: "dev"
+cache: False
+
+# Install status
+finished_common_tasks: False
+finished_db_tasks: False
+finished_web_tasks: False
+finished_mw_tasks: False
+finished_dev_tasks: False
+finished_installation: False
+
+# Apt proxy:
+# Uncomment this option and change the ip address after installing apt-cacher-ng
+# on your host to avoid downloading the same files again and again.
+#apt_proxy: host ip address
+
+# Show PHP errors
+debug: False
+
+# Sets IP for the Virtual Machine
+private_network_ip: "192.168.33.10"
+
+# Hostname to use for the development server
+hostname: "hitchwiki.test"
+
+# Setup server using self signed SSL certificate
+setup_ssl: False
+
+# Set `false` to use only private network IP to access the server.
+# Vagrant won't therefore modify your `/etc/hosts` file neither.
+hostmanager_enabled: True
+
+pkg:
+  helper: [ unattended-upgrades, composer, tmux, nodejs, npm, python-pip, vim, curl, git, unzip, zip, imagemagick, build-essential, python-software-properties, fail2ban ]
+  db: [ mariadb-server, software-properties-common, python-mysqldb ]
+  web: [ apache2, phpmyadmin, php7.0, libapache2-mod-php7.0, php7.0-mysql, php7.0-curl, php7.0-gd, php7.0-intl, php7.0-imap, php7.0-mcrypt, php7.0-pspell, php7.0-recode, php7.0-sqlite3, php7.0-tidy, php7.0-xmlrpc, php7.0-xsl, php7.0-mbstring, php7.0-opcache, php-memcache, php-pear, php-imagick, php-apcu, php-gettext ]
+  mw: [ apt-transport-https, parsoid ]
+
+dir:
+  root: /var/www
+  conf: /var/www/configs
+  script: /var/www/scripts
+  pages: /var/www/scripts/pages
+  settings: /var/www/configs/settings.yml
+  wikifolder: wiki
+  wiki: /var/www/public/wiki
+  mwconf: /var/www/configs/mediawiki.php
+  dumps: /var/www/dumps
+  phpmyadmin: /usr/share/phpmyadmin
+
+db:
+  server: mariadb
+  host: localhost
+  username: root
+  password: root
+  database: hitchwiki_en
+  prefix:
+  phpmyadmin_password: autobahn
+db_credentials: "-u {{ db.username }} -p{{ db.password }}"
+
+# Apache
+webserver_user: www-data
+webserver_group: www-data
+
+hw:
```

Comment:
Let's have this as "mediawiki" instead of "hw" for clarity.

Reply:
got me again being lazy :)

what about this structure:
```
dir
  root
  ...

ansible
  environment
  user
  status
    db
    mw
    ...
  version
    apache
    npm
    ...
  
vagrant
  private_network_ip
  hostname
  hostmanager_enabled

db
   ...

hitchwiki
  protocol: http|https # => used by ansible to (de)active transport security
  smtp
    ...
  spam
    ...
  mapbox
    ...
  geonames_usernames: hitchwiki{,2,3}
```